common/shlibs

```diff
@@ -22,6 +22,9 @@ libm.so.6 glibc-2.32_1
 libpthread.so.0 glibc-2.32_1
 librt.so.1 glibc-2.32_1
 libdl.so.2 glibc-2.32_1
+libm4ri-0.0.20200125.so m4ri-20200125_1
+libm4ri-0.0.20200125.so m4ri-devel-20200125_1
+libm4rie-0.0.20200125.so m4rie-20200125_1
```

Comment:
Why is m4riE here?

Reply:
Otherwise I get
```
=> WARNING: m4rie-20200125_1: libm4rie-0.0.20200125.so not found in common/shlibs!
```

Reply:
Please add the sonames to the end, not right between our glibc linkers...